docs-ref-conceptual/install-azure-cli-windows.md

```diff
@@ -39,10 +40,14 @@ To learn more about different authentication methods, see [Sign in with Azure CL
 
 [!INCLUDE [uninstall-boilerplate.md](includes/uninstall-boilerplate.md)]
 
-Uninstalling can be done by running the MSI again, and choosing the "Uninstall" option.
+You uninstall the Azure CLI from the Windows "Apps and Features" list. To uninstall:
 
-> [!div class="nextstepaction"]
-> [Download the MSI installer](https://aka.ms/installazurecliwindows)
+| Platform | Instructions |
+|---|---|
+| Windows 10 | Start > Settings > Apps |
+| Windows 8<br/>Windows 7 | Start > Control Panel > Programs > Uninstall a program |
+
+Once on this screen type __Azure CLI__ into the program search bar. The program to uninstall is listed as __Microsoft CLI 2.0 for Azure__. Select this application, then click the `Uninstall` button.
```

Comment:
`Microsoft CLI 2.0 for Azure` is invalid as it has been changed to `Microsoft Azure CLI` by https://github.com/Azure/azure-cli/pull/8580 on Feb 20, 2019.